src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item"></div>
+      </div>
+      <div class="stars stars--5">
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+        <div class="stars__item stars__item_yellow"></div>
+      </div>
```

Comment:
make it work like this 
![image](https://user-images.githubusercontent.com/99331793/201306571-84783c78-12f5-45bc-88ec-e70b0e36c450.png)